introduction.tex

```diff
@@ -1,14 +1,19 @@
+This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, in order for it to gain traction. We recommend utilizing proof-of-burn in order to allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy cryptocurrency.
+
+We wish to allow users to exchange a legacy cryptocurrency for a new cryptocurrency. We call the former the \emph{source cryptocurrency}, and the blockchain this cryptocurrency is based on the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
 
 \noindent
 \textbf{Workflow.}
-A user who wishes to acquire a target cryptocurrency first has to form a specially crafted address called the \emph{burn address}, which encodes among other things his receiving address on the target blockchain. The user then sends an amount of the source cryptocurrency to that address, essentially burning this amount in the source blockchain. He then proceeds to submit a proof of this burn on the target blockchain, where the burn is verified and the user is credited an equivalent amount of target cryptocurrency on his receiving address. The equivalent amount can be determined based to the macroeconomic model of the target cryptocurrency and is beyond the scope of this work. The verification of the proof of the burn can happen in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
+A user who wishes to acquire a target cryptocurrency first has to form a specially crafted address called the \emph{burn address}, which encodes among other things his receiving address on the target blockchain. The user then sends an amount of the source cryptocurrency to that address, burning this amount in the source blockchain. He then proceeds to submit a proof of this burn on the target blockchain, where the burn is verified and the user is credited an equivalent amount of target cryptocurrency on his receiving address. The equivalent amount can be determined based to the macroeconomic model of the target cryptocurrency and is beyond the scope of this work. Upon burning her coins the user can generate a proof-of-burn which is a string which can be verified by any interested party given the metadata. The verification of the proof of the burn can happen in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
```

Comment:
Be consistent with the gender of the user throughout the paragraph. Use she/her.